src/GraphQL/Builders/ISchema.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace GraphQL.Builders
+{
+    public interface ISchema
+    {
+        void RegisterRepository<TRepository>(TRepository repository)
```

Comment:
Not sure I'm keen on this repository stuff, it seems like this treats the Schema as a Container.  Can you give a use case / example?  Perhaps these could be added as extension methods in your own project if this is a pattern that you want to use?